proto/projects/project.proto

```diff
@@ -43,4 +43,9 @@ message ChainPolicy {
 message ProtoDeveloperData {
     string projectID = 1;
     string vrfpk = 2;
-}
\ No newline at end of file
+}
+
+// map subscription to a list of its projects
+message Subscription {
```

Comment:
my concern was whether it could happen that a (subscription) key is a prefix of another (subscription) key. I don't know if that's possible -and if so, probably very hard to create intentionally- but if it is then I believe we must use an extra map.

Reply:
actually, an alternative is to force a delimiter after the subscription key (in the current scheme), and then will eliminate the risk of unintended prefix issue. (Similar to how we did so in FixationStore). That's indeed better than adding the extra map.

Reply:
From bech32 spec at https://en.bitcoin.it/wiki/BIP_0173#Specification: it as two parts, the latter may contain alphanumeric characters only. Projects currently use "-" (to join the subscription address to the project name), which suffices to prevent prefix collisions with addresses.